[Docs] - add corr_interactive_plot example

m-marqx opened the pull request:
This pull request will add an example of `corr_interactive_plot` in `readme.md` and `/examples` directories.

---

After posting this example, I noticed one thing: when the user sets the `split` values with `"pos"` or `"neg"` the usual method to calculate the color scale plots different colors compared to `corr_plot`, this is a problem? If it's an issue, I can add the following code after calculating vmax and vmin in `corr_interactive_plot`:
```python
vmax = -vmin if split == "neg" else vmax
vmin = -vmax if split == "pos" else vmin
```

This fix should address the color problem.

akanz1 reviewed (dismissed):
Great, thank you for the update. I leave it up to you to decide about the colors, whichever you think is more suitable. I'll run and test everything before creating a release so if it bothers me then i can still make a change :)

m-marqx commented:
I've fixed the colors and updated the readme.md. I think this fix would be more suitable if the color scale matches corr_plot, as the main idea behind corr_interactive_plot is to be very similar to corr_plot but with interactive features.
